src/Tests/AttachmentResourceTest.php

```diff
@@ -125,7 +125,7 @@ public function testHead() {
     $attachment_info = 'field_test_file/0/' . $this->files['1']->uuid() . '/public/' . $this->files['1']->getFileName();
     $this->httpRequest("$db/" . $this->entity->uuid() . "/$attachment_info", 'HEAD', NULL);
     $this->assertResponse('200', 'HTTP response code is correct.');
-    $this->assertHeader('content-type', $this->files['1']->getMimeType());
+    $this->assertHeader('content-type', 'text/plain; charset=UTF-8');
```

Comment:
Do we need to use `text/plain` here because all attachments are base64 encoded into text?